ArticleTemplates/assets/scss/modules/_advert-slot.scss

```diff
@@ -144,6 +144,10 @@
         }
     }
 
+    .advert-slot__wrapper--1 {
+        height: 300px;
+    }
```

Comment:
Thanks for initiating this @JamieB-gu !

The below css worked for me when displaying the advert at the agreed _320px x 320px_ size. The extra height is to account for the thin 'Advertisement' and 'AdChoices' banner within the advert content.

```suggestion
    .advert-slot__wrapper--1 {
        height: 333px;
        width: 320px;
        margin-left: auto;
        margin-right: auto;
    }
```

| Square advert restricted to 320x320 | Non-Teads advert in first slot |
|---|---|
| ![square](https://user-images.githubusercontent.com/79363218/119171922-24f9af00-ba5d-11eb-8d3b-d95e592f9623.png) | ![Screenshot_20210521-174442](https://user-images.githubusercontent.com/79363218/119173154-c6353500-ba5e-11eb-9a36-53f6e9ce469f.png) |

cc: @petternitter @shesah